### PR TITLE
chore: add spanish i18n

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Adds Spanish translations.

### CODE REVIEW

File: i18n/data/de.json [hunk 1]
```diff
@@ -28,8 +28,8 @@
   "UIKit.RichTextBody.italicButtonLabel": "Kursiv",
   "UIKit.RichTextBody.moreStylesDropdownLabel": "Weitere Stile",
   "UIKit.RichTextBody.moreStylesDropdownOptionStrikethrough": "Durchgestrichen",
-  "UIKit.RichTextBody.moreStylesDropdownOptionSubscript": "Hochstellen",
-  "UIKit.RichTextBody.moreStylesDropdownOptionSuperscript": "Tiefstellen",
+  "UIKit.RichTextBody.moreStylesDropdownOptionSubscript": "Tiefstellen",
```
Comment (montezume) [inline i18n/data/de.json]: according to Filip these were flipped...

**Comment by montezume** [comment]
maybe @jonnybel speaks 🇪🇸 ? 🤔

**Comment by montezume** [comment]
@jonnybel maria wrote them...

**Comment by tdeekens** [comment]
I think she should confirm again 😄

**Pull request merged**